scripts/prep_local_devel_env.sh

```diff
@@ -159,9 +159,20 @@ while [ "$?" -ne 1 ]; do
   oc get pods -n ${ASB_PROJECT} | grep asb
 done
 
+oc scale deployment etcd --replicas 0 -n ${ASB_PROJECT}
```

Comment:
@rthallisey No after a catasb run. It only exists if you have run the prep_local_devel_env.sh script but it doesn't cause problems if it doesn't already exist. I put this in so that you can rerun the prep script over and over again with a completely reset ansible-service-broker project.